docker/airflow/1.10.5/include/entrypoint

```diff
@@ -19,9 +19,6 @@ if [[ -n $AIRFLOW__CORE__SQL_ALCHEMY_CONN  ]] && [[ $CMD =~ ^(webserver|schedule
   while ! nc -w 1 -z "${HOST}" "${PORT}"; do
     sleep 0.001
```

Comment:
https://github.com/astronomer/astronomer/pull/517/files#diff-445ef4fcea54585dda41d282aff7723cR13-R19
do we really need this block here?

Reply:
Without it, certain places, like the CLI (docker-compose), and in a new deployment, the postgres container, or pgbouncer may not be up and accepting traffic. This prevents the containers from exiting / crashing and allows for them to gracefully attach to the db/bouncer.